src/layout/section/heroSection/homeHeroSection.tsx

```diff
@@ -0,0 +1,140 @@
+import styled from 'styled-components'
+
+import { color, componentSizes, responsiveBreakpoints, space } from '../../../_utils/branding'
+import { TextDisplay1 } from '../../../typography/display1'
+
+// The image should always be 45% of the screen size.
+const IMAGE_HEIGHT = 45
+
+// 15% of the image is for spacing above the text.
+const TITLE_MARGIN_TOP = IMAGE_HEIGHT * 0.15
+
+// 20% of the image is for the title.
+// It is allowed to overflow vertically if the text wraps on multiple lines.
+// It wont impact the rest of the layout, it will only spread over the top
+// margin and the cleared out space.
+const TITLE_HEIGHT = IMAGE_HEIGHT * 0.2
+
+// 35% of the image should be cleared out for the user to see it.
+const IMAGE_CLEARED_OUT_PART_HEIGHT = IMAGE_HEIGHT * 0.35
+
+// The 30% remaining of the image is for the bottom element.
+// It is allowed to overflow by the bottom.
+const BOTTOM_CONTENT_HEIGHT = IMAGE_HEIGHT * 0.3
+
+const StyledHomeHeroSection = styled.div`
+  position: relative;
+  z-index: 1;
+  margin-bottom: ${space.xl};
+  width: 100%;
+  /* Making sure that the container is at least the same height of the image. */
+  min-height: ${IMAGE_HEIGHT}vh;
+  background-color: ${color.white};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+type HeroImageProps = {
+  heroImageUrlSmall: string
+  heroImageUrlLarge: string
+}
+
+const HeroImage = styled.div<HeroImageProps>`
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: ${IMAGE_HEIGHT}vh;
+  background-size: cover;
+  background-position: center;
+  background-image: ${props => `url(${props.heroImageUrlSmall})`};
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    background-image: ${props => `url(${props.heroImageUrlLarge})`};
+  }
+
+  ::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    display: block;
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0.1) 30%, rgba(0, 0, 0, 0) 100%);
+  }
+`
+
+const HeroContent = styled.div`
+  width: 100%;
+  max-width: ${componentSizes.largeSectionWidth};
+  padding: 0 ${space.xl};
+`
+
+const HeroTitle = styled.h1`
+  margin: ${TITLE_MARGIN_TOP}vh 0 0;
+  width: 100%;
+  height: ${TITLE_HEIGHT}vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+`
+
+const HeroBottomElement = styled.div`
+  margin-top: ${IMAGE_CLEARED_OUT_PART_HEIGHT}vh;
+  width: 100%;
+  min-height: ${BOTTOM_CONTENT_HEIGHT}vh;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  justify-content: center;
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    align-items: center;
+  }
+`
+
+// These are custom font sizes which are only used for the hero section.
+const SMALL_FONT_HEIGHT_TITLE = 32
+const LARGE_FONT_HEIGHT_TITLE = 60
+
+const HeroTextDisplay1 = styled(TextDisplay1)`
+  color: ${color.white};
+  font-size: ${SMALL_FONT_HEIGHT_TITLE}px;
```

Comment:
TextDisplay1 holds font-size and should not be overridden on normal state

Reply:
As explained in the comment above, the font sizes we need are 32 and 60 which are not in any typography components.
Are you suggesting not to use `TextDisplay1`?

Reply:
IMHO I think we shouldn't redefine components if it doesnt follow a minimum of the base spec.
Typography elements are based on Font-Size, Weight and Color. If the other Title/Text component changes 2 properties of it, should be re-declared... 

In this case, I feel its more a `HeroTilte` with its owns properties. 
We should expose this kind of changes on Chapter for discussions since it's not the first time that happens.
I'm taking the point.

Reply:
I'll merge `HeroTitle`'s and `HeroTextDisplay1`'s styles and remove the usage of `TextDisplay1`.